addons/l10n_jp/i18n/ja_JP.po

```diff
@@ -0,0 +1,1212 @@
+# Translation of Odoo Server.
```

Comment:
I am missing a POT file. 
You should also call ```process_coa_translations()``` in your init file to make sure you have the trads.